Assignment2/polkiko-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,60 @@
+{
+  "@context": {
+    "vocab": "http://www.web.com/ontologies/vocab#",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+    "items": "rdfs:Class",
+    "data": "rdfs:label"
+  },
+  "items": [
+    {
+      "@type": "rdfs:Class",
+      "@id": "http://www.web.com/ontologies/Class01",
+      "vocab:includes": [
+        {
+          "@type": "rdfs:Sensor",
+          "@id": "http://www.web.com/ontologies/Sensor029"
+        },
+        {
+          "@type": "rdfs:Computer",
+          "@id": "http://www.web.com/ontologies/Computer101"
+        }
+      ]
+    },
+    {
+      "@type": "rdfs:Sensor",
+      "@id": "http://www.web.com/ontologies/Sensor029",
+      "vocab:hasMeasurement": {
+        "@type": "rdfs:Measurement",
```

Comment:
Should not be rdfs:Measurement, since it is not describe in RDF Schema, but I accept it.